### PR TITLE
Dead station

### PR DESCRIPTION
**Pull request by SergeiKoralev**
## About The Pull Request

DeadStation is designed for exactly what the name implies, deadpop. It's focused around drawing people as quickly as possible into central/core areas of the station and making sure that players are able to see one another. On top of that there is a whole ton of maintenance space for you all to do evil things to one another. Primary features include... 
* multi-deck configuration that was included in design, meaning it shouldn't feel haphazard or ramshackle mechanically.
* A unique supermatter engine configuration
* Service-centric layout, encouraging passage around or through the bar/kitchen.
* Centrally located science department to encourage additional traffic in core areas of the station.
* Upper deck usage for ancillary department areas such as xenobiology, toxins, nanite lab, and chemical factory.

## Why It's Good For The Game

There are low-population maps, but for the most part they don't really focus on encouraging traffic around the station so you can see other players. In theory this should help with that. It also follows a more traditional footprint than Kilo or Pubby does. It also adds a true designed and built multi-z map, providing additional diversity from existing maps. Extensive maintenance space also provides plenty of areas for crime to happen even with a design that encourages player interaction.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

</details>

## Changelog
:cl:
add: Adds a new station (Deadstation)
/:cl:

### CODE REVIEW

**Comment by SergeiKoralev** [comment]
Alright, just to be clear, the current status of this is that it is ready for a play-test. It is, currently, as complete as I can make it on my own without in-depth understanding of every job (or the patience to test it all myself).

File: _maps/deadstation.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    "map_path": "map_files/Deadstation",
+    "map_file": ["Deadstation.dmm", "Deadstation2.dmm"],
+    "shuttles": {
+	    "emergency": "emergency_kilo",
+	    "ferry": "ferry_kilo",
+	    "cargo": "cargo_corg",
+	    "whiteship": "whiteship_meta"
+    },
+	"traits": [{"Up" : 1, "Linkage": "Self"}, {"Down": -1, "Baseturf" : "/turf/open/openspace", "Linkage": "Self"}]
+}
```
Comment (mcmeiler) [inline _maps/deadstation.json]: Newline
```suggestion
}

```

**Comment by MNarath1** [comment]
> _maps/map_files/DeadStation/Deadstation.dmm:10963:error: invalid variable name: harm_intent_damage
>WARNING: Found potential invalid file reference to _maps/map_files/Deadstation/Deadstation.dmm in _maps/_maps/deadstation.json
>WARNING: Found potential invalid file reference to _maps/map_files/Deadstation/Deadstation2.dmm in _maps/_maps/deadstation.json

**Comment by MNarath1** [comment]
Eh its actually normal for the hos spider to be a bat juust remove the single var on it that is long not a thing anymore but somehow still keeps showing up

**Comment by MNarath1** [comment]
En short harm intend damage

**Comment by AnCopper** [comment]
image
image

**Comment by SergeiKoralev** [comment]
> image image

there were already pictures in the pictures and video dropdown of the PR.

**Comment by Archanial** [comment]
With our current map rotation we can't have this enabled.

And if we can't have this enabled what's the point of having the map?

**Comment by williamcll** [comment]
Multi-Z wires still don't work, power on the lower deck can't be transferred to the upper deck yet.

**Comment by SergeiKoralev** [comment]
> With our current map rotation we can't have this enabled.

Why can't we?



> Multi-Z wires still don't work, power on the lower deck can't be transferred to the upper deck yet.

They work. I tested them.

**Comment by williamcll** [comment]
> > With our current map rotation we can't have this enabled.
> 
> Why can't we?
> 
> > Multi-Z wires still don't work, power on the lower deck can't be transferred to the upper deck yet.
> 
> They work. I tested them.

I think he means that there's too many maps on rotation? Because Tubestation and Corgstation are being merged first.

**Comment by PowerfulBacon** [comment]
We have a lot of map PRs right now, so this might take some time to get through

File: config/maps.txt [hunk 2]
```diff
@@ -41,6 +41,14 @@ map deltastation
 	votable
 endmap
 
+map deadstation
+	minplayers 0
+	maxplayers 25
+	votable
+endmap
+
+#Does not support fastmos
```
Comment (PowerfulBacon) [inline config/maps.txt]: Any point for this message?

Comment (SergeiKoralev) [reply config/maps.txt]: Honestly, unless a fastmos PR comes along there isn't any point to that. If you want me to remove it I can.

Comment (PowerfulBacon) [reply config/maps.txt]: Yea, probably best since all our maps use the new firelocks if that's what you mean by does not support fastmos

Comment (SergeiKoralev) [reply config/maps.txt]: removed in 5aed385

**Comment by PestoVerde322** [comment]
Ok I've checked the map, and first thing that was very evident is that the Z-layer is in two different maps, while this is a system used in other places, Beestation can support maps with Z layers, you can do this by using StrongDMM and set the map size to 2Z (and also use StrongDMM2, which has proper multi-Z support).

Second off, this:

immagine

I suppose this is a leftover from the FastMos airlock system, but really having two firelocks like this is nothing but redundant, I strongly suggest to do as any other common map do, which is to shove the firelock under the airlocks.

immagine

Any Virology trash will hit the wall over there, not so biohazard safe I'd say.

immagine

All the assistant spawns are localized within the bar, suggest to move them around really otherwise not only it will feel unnatural, but in the (rare) case if this map suddenly goes highpop, and many people decide to sign as an assistant, there's basically a Battle royale light to get all the goodies of the tech storage.

immagine

This maint "corridor" feels weird, suggest to actually use a legit corridor and distribute the maint rooms around (or to remove them altogether, considering the station is on multi-z)

immagine

also those APCs around maint will act weird with wiring.

immagine

the Gravity generator could be relocated somewhere else, where it is right now it can be prime target for nuke ops.

This comes down a bit more to personal taste, but a lot of those empty gaps kinda feel... pointless in a station environment, where they could be filled instead with other stuff, and also a lot of ladders are just weirdly located:

immagine

immagine

you could place some stairs instead and some of the rooms, like the tech storage (which currently sits in a quite isolated location next to the server room, which by the way it's WAY too big for any station) and, speaking of, i am not 100% sure how mechs would get up or down a floor in this station considering they can't use stairs.

immagine

Telecomms is quite thick, but those internal double walls don't really make much sense if you can just breach a window up there and get in easly, and speaking of double walls:

immagine

avoid those settings, fill them with maint junk instead, and really, there's an overabundance of random maint in the second floor, consider trimming them down a little or add some more connected rooms.

immagine

Perkele, a sauna center!

Also many cables seems to use an invalid icon state, Suggest to just use an existing map for reference on cables.

immagine


Overall this station might need plenty of tweaks, but it might be interesting to see a tight map with lots of maint rooms.
Also update your game version, the most recent version i got is quite dated.

**Comment by SergeiKoralev** [comment]
Alright, responding now, fixing later. 

- Differentiation in map files is intentional and there is realistically no significant difference. I also couldn't get a single .dmm with Z2 to work, so I did what NSV does (This will be a theam).
- The firelocks are an intentional alteration on Kilo's normal configuration. They provide a better counter for fire and high heat and don't crush you when you need to go through them. In short, they're placed that way to provide an emergency "airlock." It also makes firelock crushing easier for mean AI's, so it's not all positive.
- Why will maint APCs act weird with the in-line wiring?
- Do you have any suggestions for relocating the grav generator?
- The empty gaps above the bar either look down on space, or on the bar. 
- the window north of tcomms is in engineering-restricted maintenance which is, if I remember correctly, only accessible from either engi or space. So it's a struggle either way.
- the invalid icon cables are vertical nodes, which haven't had a sprite generated. 



TO DO
- [x] move viro disposals a bit. (looping z makes this a bit pedantic but you have a fair point.
- [x] Make more assistant spawns around the map. (or let them brawl.)
- [x] Figure out a way to make maint south of SM less 3x3y
- [x] trim back random maint in upper z
- [x] merge from mastere
- [x] add more stairs in somehow?
- [x] Test mechs going up/down stairs.

**Comment by PestoVerde322** [comment]
> Why will maint APCs act weird with the in-line wiring?

It's something i've experimented on the early version of fland (pre-release) and i noticed that the rats would chew on the main cable making the APC however practically "invulnerable" to rat attacks, as one side is always guaranteed to get power (plus engineers don't really like that configuration so).

> Do you have any suggestions for relocating the grav generator?

Either on the second floor's hallway or in place of the pod next to the science shuttle.

The rest I guess then it's good to go

**Comment by SergeiKoralev** [comment]
> It's something i've experimented on the early version of fland (pre-release) and i noticed that the rats would chew on the main cable making the APC however practically "invulnerable" to rat attacks, as one side is always guaranteed to get power (plus engineers don't really like that configuration so).

I'm going to categorize this as not an issue. The mouse might target the wire going under first, but a second would still get the node and there's still the added damage of no power going through next. 



> Either on the second floor's hallway or in place of the pod next to the science shuttle.
> 
> The rest I guess then it's good to go

I think I'm going to leave the grav gen in place. For most people, including nukies, it's pretty annoying and more a target of convenience. I'm not that concerned. If it winds up being a major issue I'll alter it.

File: code/game/machinery/lightswitch.dm [hunk 3]
```diff
@@ -10,6 +10,23 @@
...
+/obj/machinery/light_switch/directional/east
+	pixel_x = 27
+	pixel_y = 7
+
+/obj/machinery/light_switch/directional/west
+	pixel_x = -27
+	pixel_y = -7
+
+/obj/machinery/light_switch/Initialize()
 /obj/machinery/light_switch/Initialize(mapload)
```
Comment (PowerfulBacon) [inline code/game/machinery/lightswitch.dm]: 🤔

**Review by PowerfulBacon — COMMENTED** [review]
The top floor and bottom are 2 entirely different map files? Shouldn't they be a single map file and use the z value in that map file, so the mapping tools display properly?

**Comment by PowerfulBacon** [comment]
I definitely can feel this map having the same problems as multi-corg, the main floor is on the bottom with the top being mostly maintenance, allowing for threats that you can't see.
The maints floor should be underneath the floor with all the stuff on it.
It feels like a single-z map with an extra floor added on top, since almost everything is on 1 floor.

There is a ladder that just goes straight into security
image

The map feels a little unpolished, it seems to lack detail in areas. For example, this hallway has no decals and feels a little lacking
image

This staircase leads straight into maintenance, why would a staircase go to maints.
image

Some of the random maints are made wrong, the doors should be in the middle.
image

There are also some missing doors.
image

This maintenance dead-end is a little annoying to get stuck in
image

Medical has no cryo tubes.
Seems there are no cryo pods either.

The armoury is absolutely huge.

My main critisism is the top floor being poorly utilised, being entirely maintenance. The problem with the main floor not being the top floor is that you face threats that are completely impossible to fight since they are invisible to you.
I think in order to be a good map, a good first step would be to swap the top and bottom floors around.

**Comment by PowerfulBacon** [comment]
Detective has no office

**Comment by ghost** [comment]
I don't think this map has ~~any~~ enough cryopods, I can't find any in-game besides the one in perma and I don't see any more on the screenshots.

**Comment by Miliviu** [comment]
From what I have seen the emergency maintenance access does not work on the secondary floor!

**Comment by PowerfulBacon** [comment]
After playing blob on this map, I can say for sure there should not be an entire floor of maints that is almost impossible to access apart from some easilly destroyable ladders.

**Comment by Pirill** [comment]
Service lathe can be accessed from maints on the top floor via ladder

Weird SM mapping, I'm not even sure those connect right;
image

Weird powernet issues and confusing wiring scheme. The use of yellow and red wires seems a bit mixed up in Engineering. Also atmos wasn't even powered at the start for some reason until I exposed some wires via floor tiles. Not sure if this was a weird coincidence or a bug;

Engineering closets are locked inside secure storage, making it inconsistent with other maps, especially since secure storage has Engi access here, not CE.

Also no rapid cable layer in Engineering unless I missed it.

Weird stair ladder combo near dorms
image

Dorms could use a washing machine;

EVA has no RCD;

Upper level overall just needs more polish. It's a mass of maint rooms with little to no landmarking.

File: code/game/machinery/lightswitch.dm (hunk 3 above)
Comment (SergeiKoralev) [reply code/game/machinery/lightswitch.dm]: That must've changed when I made directional versions of light switches. I'll remove that line shortly.

**Comment by Vasily2013** [comment]
> The top floor and bottom are 2 entirely different map files? Shouldn't they be a single map file and use the z value in that map file, so the mapping tools display properly?

I attempted to get it working with a single .dmm but was having trouble, so I went to using the method NSV utilizes, which works perfectly fine. 

TODO

- [ ] Remove that random line from lightswitch.dm
- [ ] More Decals
- [ ] Try to tweak random maint rooms that don't meet door requirements.
- [ ] Correct missing doors in some maint rooms.
- [X] Cryo pod added to cloning room.
- [X] Added stairwell in evac.
- [X] Added cryo pods above evac.
- [ ] Make detective office
- [ ] Check emergency maint access functionality on upper deck. 
- [ ] Add even more stairwells
- [X] Electrical and welding supply lockers are no longer in the engineering warehouse.
- [X] Cable layer added to engineering.
- [X] Added washing machines to dorm rooms.
- [X] RCD crate added to EVA


NotABug

- Maint floor can't be under the upper deck in current configuration due to the placement of the AI's core and upload. It would unreasonably expose them to attack from above. 
- The ladder in sec storage leads up into a security-only maint area, and as such I felt secure in placing that ladder there without significant consequence. 
- The staircase is literally only there to provide mech access up and down. As you can see in the screenshot, the mech bay is right next to the stairs.
- Medical maints has medical access restriction starting next to med storage. I, personally, wouldn't qualify it as a dead end. 
- Access to the service lathe isn't a major concern in my opinion. This map is designed for low-pop, and thus access to service is expanded already. Access to custodial isn't a major addition. 
- Red and yellow wires aren't mixed up in engineering. Yellow is power outputted from the SMES. Red is direct from power sources. Many areas in engineering are powered directly by power sources such as the turbine or rad collectors.
- Armory is as small as I can make it and still be consistent with the layout of other armories.

**Comment by PowerfulBacon** [comment]
> Maint floor can't be under the upper deck in current configuration due to the placement of the AI's core and upload. It would unreasonably expose them to attack from above.

You could move it, its more worth making the entire map better than just the AI core.
The #1 thing I learnt from Multi-z corg and the reason it was a complete design failure was because I added the second floor on the top. This top floor was basically unused apart from a few small departments like botany and some science stuff. The problem was this gave direct access to everywhere else through the roof, which you cannot fight. There are a lot of parallels between this map and multi-z corg, so I would recommend trying to avoid the same mistakes and issues that corg faced.

**Comment by Vasily2013** [comment]
- Wiring in the engineering windows is to make it look like the windows are hooked up to the main grid when they're actually hooked up to the engine. This is a devious trap to prevent greytiders from  breaking in to get insuls while at the same time enabling greytide to set the engine up easier if there is no engineering staff to set up the engine. 
- Stairs near mech bay work and have been tested. 
- added a confessional in chapel basement
- electrified HOP windows.
- Suggestions on perma re-shaping would be appreciated. I am borg.
- Padding removed, tables moved to fill in area.

- [ ] reshape area under bar into a big shower/dorm room type thing.
- [ ] Flesh out detective antechamber.
- [ ] mystery room was previously right above the CMO's office. Refactor area below CMO's office.

**Comment by Vasily2013** [comment]
Alright, so, now for pictures I guess? 

Central lower maint
image

HOP office
image
Science Hub / Nanite Lab
image

**Comment by Vasily2013** [comment]
Removed some volume from the explo airlock. 
image

Removed nubs and spare Rwall. Previously they were on top of the arrivals docking ports as "ceiling". 
image

**Comment by PestoVerde322** [comment]
> * Wiring in the engineering windows is to make it look like the windows are hooked up to the main grid when they're actually hooked up to the engine. This is a devious trap to prevent greytiders from  breaking in to get insuls while at the same time enabling greytide to set the engine up easier if there is no engineering staff to set up the engine.

As much i hate greytide, that may lead to a lot of Ahelp from salty assistants who claim that some engines hotwired the glass for trolling purposes, consider non-hotwire electrified glass.

immagine

This maint hurt my soul, please consider reshaping with some hallways.

immagine

immagine

what's with those empty rooms?

immagine

Why the R-walls here?

immagine

What's the reason for this maint here to exist?

immagine

Still not sure how this whole maint block is accessible but from EVA, what happens if a gimmick job spawns there?

**Comment by Vasily2013** [comment]
- I don't think the engineering windows being hot-wired by default is going to be an issue administratively.
- The three empty rooms in the top screencap are directly below the brig cells (previously above) and thus were made inaccessible. 
- The area near t-comms used to be directly above the CE's office, and there wasn't any viable usable space there for the t-comms office. I can put another desk there, though. 
- The interior R-wall along med maint is to separate normal maints from medical restricted maint.
- The maint next to sec/HOP office is for two things: First, so some clever asshole can use it to break into one or the other, and second, to provide maints near both departments in the event of radstorms. 
- [ ] ~~then the gimmick dies in solitude~~ re-jigger atmos basement. 
- [ ] Hallway in the central south basement maybe?

**Comment by Crossedfall** [comment]
Reloading checks, don't mind me.

**Comment by Vasily2013** [comment]
> rather cursed Piping

OH COME ON IT'S NOT THAT BAD!

**Comment by Vasily2013** [comment]
> put the virology to the Zams' standard

Where can Zams' standard be found?

**Comment by PestoVerde322** [comment]
> Where can Zams' standard be found?

Thanks gmail to not sending me a mail about a received reply!

anyhow, it's in box, meta, delta and pubby, just mimic what's in there

**Comment by PowerfulBacon** [comment]
Ready for TM?

**Comment by Vasily2013** [comment]
> Ready for TM?

it should be ready for a TM to see what breaks.

**Review by PowerfulBacon — DISMISSED** [review]
Approving for testmerge

**Comment by itsmeow** [comment]
fix MC please

**Comment by itsmeow** [comment]
Does this use mutliz cables or multiz power adapters? It should be using multiz cables similar to glow. To get multiz cables just use dirs 16 and 32 in the icon_state of the cables

**Comment by Vasily2013** [comment]
> Does this use mutliz cables or multiz power adapters? It should be using multiz cables similar to glow. To get multiz cables just use dirs 16 and 32 in the icon_state of the cables

it uses multi-z cables.

**Comment by Littlecasino** [comment]
bump 
new map for low pop would be cool

**Comment by Vasily2013** [comment]
> bump new map for low pop would be cool

it would be.

**Comment by PowerfulBacon** [comment]
image

This maint room should have the ladder in the center

image

You can enter this room via the maints without access

image

This room is very weird

**Comment by PowerfulBacon** [comment]
image

Arrivals is broken

**Comment by PowerfulBacon** [comment]
Science and perma have no cams

**Comment by PowerfulBacon** [comment]
image
newcaster in chapel

**Comment by PowerfulBacon** [comment]
image

**Comment by PowerfulBacon** [comment]
image

**Comment by PowerfulBacon** [comment]
image

These firelockos bug out the appearance of the table

**Comment by aramix273** [comment]
in the prison sanitarium, the ladder seems to not be properly set up, the upper floor just has some open space
image
image
image

**Comment by aramix273** [comment]
> image
> 
> This room
speaking of this cryo room, to the bottom left of the image you can see what I believe is supposed to be a staircase
on the lower floor, there is an appropiate set of stairs leading to it, but on the upper floor it seems to be blocked by plating
uncertain if this is intended

**Comment by Vasily2013** [comment]
Oh my gods, feedback. Time to look at this again!

**Comment by Vasily2013** [comment]
> image
> 
> This maint room should have the ladder in the center
> 
> image
> 
> You can enter this room via the maints without access
> 
> image
> 
> This room is very weird

Ladder moved to right door position because if I put it in the middle of the room the ladder below will be in the middle of the hallway.

Added sliding door to provide access restriction.

Removed random maint spawner from weird room.

**Comment by Vasily2013** [comment]
- [ ] Fix arrivals (again)
- [x] Fix newscaster in chapel
- [x] Fix pews
- [x] Put bar tables on top in bar.
- [ ] Verify decks are in right orientation

Other remarks (not currently able to identify issue in map editor)
- "this room" above evac has open space where the plating should be.
- NTNet Quantum Relay is in Tcomms, upper deck port side. Above SM.
- Door labeled kitchen as CE's door reads as "Chief Engineer's Office" in editor.
- Brigbay open space problem not replicated on local branch. I think your map is upside down.

**Comment by itsmeow** [comment]
Implement desk bells PR at some point btw

**Comment by itsmeow** [comment]
Also have you done the tilening?

**Comment by benbot16** [comment]
1. Exploration dock isn't an airlock
2. Explorers can't access the exploration dock.

**Comment by itsmeow** [comment]
Borgs can't use ladders... and there's lots of ladders, namely in their spawn location that they cannot escape from.
No NTNet connection on bottom z looks like
Lots of empty maintrooms on the east end
upper floor hallways desperately needs some decals, it looks very dead
stylistically I would prefer trimlines over the square floor decals
HoP line is very ugly, it should have a different floor or something

too many firelocks, it looks very off. they should be under the doors
image

image

No camera outside armory
No easy way into tcomms for regular engineers, also tcomms has double rwalls which is a bit much imo

This floor looks bad

image

Plumbing room is FAR too large and empty

Toxins airlock does not work at all and has no igniter inside

AI has no cams on toxins chamber

No access on this maints door as an engi
image

No lights in the dorms shower room
As far as I can tell there is no bathroom with a mirror / toilets

Cryo is really big for no reason and in a really big spot. It doesn't need to be so large and should be in an obvious location.

This ladder/stair combo is useless

image


SM delam.. is DEVASTATING. fucking awful. not just all of engineering and half of atmos, but robotics, the main hall, parts of cargo, a massive part of maints.

It literally ruins the round, might as well call shuttle since there is NO recovery.

Also, the upper space level doesn't use openspace, it should be openspace not regular space. Very confusing when you can't see below you. some areas will need roofs added to accommodate this

**Comment by itsmeow** [comment]
There are evac signs by sec/bridge that point to a pod, don't do that. Evac should point to the shuttle only.

**Comment by itsmeow** [comment]
image
kilo shuttle intersects station

**Comment by itsmeow** [comment]
Airlocks don't line up well either
image

**Comment by itsmeow** [comment]
There is no theatre with the clown's megaphone

**Comment by Fronsis** [comment]
Captain office needs a safe for the spare id..  Missing nuke disk pinpointer, hand tele should be there, ideally try to get an idea on how captain's office are made.
Screenshot_9
Brig Phys was correct, even if it's skeleton crew access they shouldn't be able to access there
Screenshot_10
To also add that there's no security hardsuits just eva ones.
not sec hardsuit
For the love of god PLEASE put a computer with operation program, it was VERY needed and with dead-pop at least give them that considering how research is going to most likely be slow as fuck.
operating computer medbay
Explorers can't even access to their ship since that airlock is labeled exterior, they had to deconstruct it
Screenshot_11

Escape shuttle badly placed doors/walls/airlocks
Screenshot_12

**Comment by itsmeow** [comment]
There is no public tool storage

**Comment by itsmeow** [comment]
No public lathes in cargo or science

**Comment by itsmeow** [comment]
Science needs a power cell charger

**Comment by itsmeow** [comment]
image
this seems to have space lighting

**Comment by TransAggressive** [comment]
Issue noticed as engineering cyborg:

AI satellite does not have any way to leave other than ladder. Since borgs can spawn in there, they have no way to leave, other than removing floor to fall a level; borgs apparently cannot climb ladders? (See below)

As MD:

Medical seems to be missing some, frankly, necessary things. There is no cold gas for cryo, none of the blue goo for cryo (or medical wrench), there are two unwrenched frames beside beds through medical that seem to be intended for operation computers? Also, I think there's no crew monitor console.

Also, for some reason, even when playing a person with hands and feet and other fleshy bits I was still unable to climb ladders. I tried the 'move up/down' in the IC menu, as well as hitting F when next to (orthogonally and diagonally) ladders; every attempt announced 'there is something in your way', I believe. Due to the pervasiveness of this, I am sort of assuming the fault might be in my settings.

**Comment by Gilgaxx** [comment]
I am making a big ol' post!

1. Kat mentioned the space lighting above viro:
dreamseeker_bJsgvMSxXT

2. But what about the one between the bridge and the hallway?
dreamseeker_oX36yxMWqH

3. Why are these walls in maints (above tcomms) reinforced if all you need is maints access to get past them anyway?
dreamseeker_By4yD4Hftr

4. Camera on the steps hehe
dreamseeker_NYOSaOB6Yd

5. Outside the brig entrance, this evac sign actually points to a pod:
image

6. These big airlocks are kinda wack. Not only do they take forever to refill, but come on, if you're gonna make it this big, atleast add an o2 locker or two?


7. Triple reinforced wall stinky (but also the north part is just one so what's even the point?)
dreamseeker_l77GzakScR

8. Also what's the point of these? They're not even reinforcing anything.
dreamseeker_YsIJAD6s1F

9. The entire north maints (and the det room) on the top floor is only accessible through the ladder in the security office without breaking stuff.
dreamseeker_efRlWi8R6Y

10. I mean I get being different from other maps, but autolathes not being public? While it's maybe good that not every clown and assistant can use them willy nilly, maybe add one in engineering/medical.
dreamseeker_9JpvVnZq0i
dreamseeker_OOhdFC5OPf

11. Judging from the impossible corner sofas (which I think is funny, ngl), this isn't a randomly spawned room, but the multi z level pipe adaptors are just kinda clipping on stuff:
dreamseeker_UBMnUMcsHP

12. What's the point of this? :v
dreamseeker_vU2bJEF2KE

13. ln game, in the toxins lab, I saw someone have to use a remote signaller to igniter assembly to ignite the gas inside the toxin's chamber, so maybe adding an igniter with a button outside would be nice x)
dreamseeker_SXqF7gqQvM

14. I mean I get protecting the cargo shuttle from space debris, but this looks... odd:
dreamseeker_rEIRnjdNRz
I think things like this (Metastation's new toxin lab) has a better example of what space dust protection means
image

15. Atmos. Oh god. I'm not sure if it had been modified when I took the screenshot, but it looks a bit messy.
dreamseeker_46UouXdzO5
There's things like how you can hide the real distro and waste pipes under tiles with paint like pubby's atmos:
image
Maybe colouring the "recycling loop" green instead of the same red as waste would help?
There's a few other odd things like, where does this pipe go in the turbine?
image

16. Why is there a wire going left, only to turn down into a dead end in the CE's office?
dreamseeker_zPytQ7hXNt

17. Speaking of the CE's office. Why is it connected directly to the SM power (unlike the emitters)? This wouldn't be too big of a problem if the only Z-level tcomms relay didn't stand right in the CE's bedroom. This means that setting up solars instead of the SM will result in half the station not having comms.
dreamseeker_V7TeeJPRRV

18. I noticed something odd as well. I took the following two photos on top of eachother (I took one, moved up and took another):
dreamseeker_oAF040UcLp
dreamseeker_1d72PKGDfV
I'm not sure if this is something I should be reporting here or to Kat for the multi-z-level update, but you can't see the things under it in the second photo.

19. Grav gen room has no cameras, meaning that the AI can't restart it's APC when theres a powernet testing event:
dreamseeker_EoOTLmGKGJ

20. Why does the Tcomms controll room have that weird corner on the bottom left?
dreamseeker_29n4ampfIW

**Final thoughts:**
This is a map with a lot of potential, I like it, but it feels unfinished. Few maint rooms are just absolutely nothing. The chem factory room is ENORMOUS. I think there's a few missing cameras but I didn't play AI so I'm not sure. But yeah, if you keep giving some touch ups, I'll look forward to playing deadstation during dead pop!

**Comment by itsmeow** [comment]
I have a few major structural issues with the map.

The upper floor generally just feels completely useless and more like a "department extension zone"

I would much prefer if there were at minimum some central hallway connecting all the disparate areas up here.

Also, the use of ladders. Ladders suck generally and should usually be replaced with staircases so borgs can use them. Also stairs allow for transport of things like lockers etc. There are VERY few staircases.

**Comment by Vasily2013** [comment]
> Implement desk bells PR at some point btw

Mind providing a PR number?

[ ] - Make explo dock an airlock
[ ] - Explorers need access to explo dock.
[ ] - Add stairs to office outside AI core.
[ ] - Verify NTnet works on bottom deck. (Fix if required)
[ ] - East side maint room filler
[ ] - Upper deck hall decals (although it is DEAD station...)
[ ] - find way to make HOP line pretty
[ ] - Add external armory camera
[ ] - adjust cargo dock to be less ugly maybe?
[ ] - Add cam to toxins
[ ] - Add igniter to toxins
[ ] - Fix toxins airlock
[ ] - add light in dorms shower
[ ] - jam a toilet and/or a mirror somewhere
[x] - Adjust evac signs to point towards shuttle dock
[ ] - Adjust shuttle dock so kilo shuttle doesn't clip. I swear it fit before.
[ ] - Find place for theater, clown/mime office.
[ ] - Add cell charger to science
[x] - fix space lighting north of viro
[x] - add medical wrench
[x] - fix newscaster in med hallway
[x] - Move camera from steps in evac
[x] - add o2 lockers to ferry shuttle dock.
[x] - remove some of the walls behind AI upload.
[x] - lathe jammed into random corner of engineering
[ ] - repaint recycling gas loop green
[ ] - Add cameras to grav gen room.
[ ] - 
[ ] - 
[ ] - 


Needs more info/investigation
[ ] - Trimlines instead of square decals?
[ ] - Critical areas do use openspace, but they look down on, mostly, more space, and therefore it looks like space. Please provide specific examples to rectify.
[ ] - Verify ladder function
[ ] - try to fix multi-deck adaptors just overlaying on things
[ ] - Investigate trying to establish a central hall on the upper deck.
[ ] - Ensure lockers can be moved up and down ladders still. (Unless this has changed then it's still doable).


not-an-issue
- Firelocks are positioned in the way they are to provide an effective means to contain fires.
- Normal engies typically don't need tcomms access.
- Tcomms defense is justified based on it being upper deck and surrounded by maint which will likely not be frequently visited.
- Plumbing room is supposed to be large and empty.
- Tech storage door has tech storage access. Not a problem.
- Find a better spot for cryo and I'll gladly move it. It's there because nothing else fits there easily. It's *Directly above arrivals and evac*. 
- Ladder stare combo not a problem. Fires may actually make it preferential to take a ladder.
- SM delam maxcap not considered major factor in design. Advise crew to not delam engine. Maintenance item closed as user error. (Also robotics is nowhere near the SM)
- Captain pinpointer located in captain's bedroom exactly one (1) door north of photo claiming it is not there.
- Station is designed for low population. Accessibility (albeit strained accessibility) of the captain's ID is intentional to permit small crews to gain required access faster than on normal stations. This is an intentional design feature. Command staff should work to secure the ID in a safe or on their person as needed. A safe can be found in the captain's bedroom. 
- ID on sec equipment room doors are commented access to sec equipment, records, etc, in addition the forensic locker access, which is responsible for the detective's office. Suggest looking at access granted by understaff scripting. 
- Security hardsuits are located in armory, as is normal-ish.
- There is, at round start, an operating computer in medbay which functions. Just because the station is made for low population does not mean I will hand you fun and convenient things. That takes away things to upgrade, which yes, takes time, but the entire game takes time.
- Lack of public tool storage intentional. Improvise, adapt, overcome. There should be plenty of maint to scrounge from.
- Lack of public laythes feature, not bug.
- Medical cryo tube has freezer located on tile north of cryo tube.
- Rwalls above (and surrounding) atmos are present because it is a slightly higher security area, even though one only needs to get through one abandoned airlock to get into the area primarily.
- reinforced section near sec office is directly above execution room exhaust. It's a security measure protecting the dock so someone doesn't just easily yoink an antag.
- restricted access to sec maint is intentional because it is directly above security. 
- area directly above SM cooling loop protects SM cooling loop (and deters sabotage) from above. Also provides additional external protection for tcomms
- random floor plating use to be roofing. Random wall doesn't stop debris, it stops idiots from going out the opposite doors on the shuttle.
- random purple manifold takes exhaust from the dual port vent and pumps it into the turbine room.
- Wire in CE's office goes up to supply power to tcomms
- wiring SM to CE's office was to prevent the tcomms SMES from drawing all the power off the grid. I think (unless it causes major problems) I'll leave it like this as a nice little engineering problem for not using the primary power supply. 
- not seeing things under you is an issue related to using openspace.
- weird little corner is because running power through walls is cancer.
- Upper deck is meant to be cold and heartless to an extent. It's intended to try and push people back to the lower deck to be social with the few other people in the round. It also provides a lot of space to do the crime.
- Staircases suck because there's no good way to atmospherically isolate them, and they take up a massive amount of space.

**Comment by itsmeow** [comment]
Desk bells: #7810

**Comment by itsmeow** [comment]
> Critical areas do use openspace, but they look down on, mostly, more space, and therefore it looks like space. Please provide specific examples to rectify.

On z2 you are using /turf/open/space as the base floor turf outside.
This means you cannot see things below you in space.

You should instead use /turf/open/openspace so that you can see things below you.

You also can't move down through /turf/open/space, only /turf/open/openspace.

Therefore on z2 all instances of space should instead be openspace.

**Comment by itsmeow** [comment]
Additionally the pinpointer should probably just be under the nuke disk rather than in another room. 

> Normal engies typically don't need tcomms access.

As a deadpop map you should really make it accessible. Usually engis do have access to tcomms, just not the monitoring room on maps like kilo.

**Review by PowerfulBacon — DISMISSED** [review]

**Comment by Vasily2013** [comment]
More TODO
- [ ] Desk Bells
- [ ] Add some way to access tcomms as a basic engineer (This will probably involve some back door in maints)

**Comment by itsmeow** [comment]
@Vasily2013 bump? I actually quite like this map and would like to see it in the game

**Comment by tiramisuapimancer** [comment]
Compiled a list of things that don't work while playing as AI on Sage round 43968 when this was testmerged: 

LOTS OF ATMOS ISSUES, especially related to areas lacking ceilings
LOTS OF POOR CAMERA COVERAGE basically everywhere
Deck one has no comms relay (!!!!!!!!) (or maybe it does? d1 comms started working after the supermatter was turned on, and engineering is hotwired to it instead of on the grid...)
some holopads are inaccessible to the AI for some reason (possibly intended behavior?)
Most Engineering department APCs aren't wired to the grid (I think they might actually be wired directly to engine output. What ??)
Toxins cooling loop is way too little to actually cool the plasma
Sci autolathe isn't public access
Sci has both a techfab and a protolathe
Toxins storage has a bomb suit locker on the same tile as a tank dispenser
no clothing vendors, possibly?
Botany has no water
No radiation shutters for the supermatter
No portable vents in engineering

**Comment by itsmeow** [comment]
This room starts spaced
image

So does here
image

Comms seem to start broken, or break VERY FAST

No air in this maints
image

Lack of tool storage makes antaggery (blood bros especially) insanely hard if you have little access, as you have no ways to get tools or access. Also if there's no heads or engi, you can't get into engi for tools TO GET INTO ENGI

wrong layout
image

**Pull request closed without merging**